db/migrate/20211128175028_add_secure_password_to_users.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-class AddSecurePasswordToUsers < ActiveRecord::Migration[5.2]
-  def change
-    add_column :users, :secure_password, :boolean, null: false, default: false
-  end
-end
```

Comment:
Deleting a migration will cause other issues. You should instead create a new migration that removes the column instead

Reply:
You got reason it's wrong.
I'll change it.

Reply:
DONE